app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md

```diff
@@ -3,114 +3,85 @@ title: Who do you want to teach
+        link_target: "https://schoolexperience.education.gov.uk/"
 ---
 
-If you want to teach a particular student group you might need to do specific training. Find out what you can do to get the right qualifications.
+It’s good to consider what kind of teacher you want to be based on your interests, skills and experience. You might need specific training or qualifications in some areas. This page explains more about the different options.
 
-You could teach:
+## Teach in a primary or secondary school
 
-- primary (ages 5 to 11)
-- secondary (ages 11 to 18)
-- disabled students and students with special needs
-- early years (children up to the age of 5)
-- further education (for example, teaching at further education colleges or sixth form colleges)
+The school system in England is split into age groups:
 
-If you want to [teach in primary or secondary schools](/train-to-be-a-teacher/if-you-have-a-degree) there are a number of ways to get [qualified teacher status](/what-is-qts) (QTS) which you will need to teach in the majority of schools in
-England.
+### Primary (ages 5 to 11)
+Children usually start primary school at 4 years old and this phase is called Reception. This is the final stage of the Early Years Foundation Stage (EYFS).
 
-You will need a degree to teach all student groups except those in further education, depending on the type of further education you are teaching.
+There are a further 2 phases, known as key stages: 
 
-## Teach disabled pupils and pupils with special educational needs
+* key stage 1 covers year 1 and 2, up to 7 years old
+* key stage 2 covers year 3 to 6 with children from 7 to 11 years old
 
-Most disabled pupils and pupils with special educational needs learn in mainstream schools.
-No matter what settings you train and work in – whether it’s mainstream schools, special schools, or both – you’ll work with pupils with complex needs.
+The teaching content is broad, covering the whole range of national curriculum subjects including English, maths, science, as well as art and design, computing, geography, history, music and physical education (PE).
 
-You can [find a course which specialises in special educational needs and disability (SEND)](https://www.find-postgraduate-teacher-training.service.gov.uk/) if you have a particular interest in this area.
+If you like the energy and curiosity of young children, and would be comfortable covering a wide range of subjects, you might prefer teaching in primary schools. 
 
-However, it’s not essential to do a course specialising in SEND. As you train and teach you’ll get the skills you need to work with disabled pupils and pupils with special educational needs. This will enable you to work in a range of settings throughout your career.
+If you’re doing initial teacher training (ITT), you could consider options for primary teaching with a subject specialism like maths or English. This will cover all of the subjects but with a focus on your subject specialisation.
```

Comment:
Does this make a bit of leap - they won't be doing ITT yet and might not know what that is?

More like 'If you are interested in training to be a primary teacher, .....'?